geps/gep-1897/index.md

```diff
@@ -149,61 +117,53 @@ certificate, we implement a metaresource named `BackendTLSPolicy`, that was prev
+* A service producer may provide TLS settings that are unacceptable to the client. For example, an `insecureSkipVerify` option. The client can override this with their own more secure settings.
+* A service may be connected from a variety of context, some requiring TLS origination and others not. Allowing client-driven control allows both of these to co-exist.
+
+The `BackendTLSPolicy` must contain these configuration items to allow the Gateway to operate successfully
 as a TLS Client:
 
 - An explicit signal that TLS should be used by this connection.
 - A hostname the Gateway should use to connect to the backend.
-- A reference to one or more certificates to use in the TLS handshake, signed by a CA or self-signed.
-- An indication that system certificates may be used.
-
-BackendTLSPolicy is defined as a Direct Policy Attachment without defaults or overrides, applied to a Service that
-accesses the backend in question, where the BackendTLSPolicy resides in the same namespace as the Service it is
-applied to.  The BackendTLSPolicy and the Service must reside in the same namespace in order to prevent the
-complications involved with sharing trust across namespace boundaries.  We chose the Service resource as a target,
-rather than the Route resource, so that we can reuse the same BackendTLSPolicy for all the different Routes that
-might point to this Service.
-For the use case where certificates are stored in their own namespace, users may create Secrets and use ReferenceGrants
-for a BackendTLSPolicy-to-Secret binding.  Implementations must respect a ReferenceGrant for cross-namespace Secret
-sharing to BackendTLSPolicy, even if they don't for other cross-namespace sharing.
-
-One of the areas of concern for this API is that we need to indicate how and when the API implementations should use the
-backend destination certificate authority.  This solution proposes, as introduced in
-[GEP-713](../gep-713/index.md), that the implementation
-should watch the connections to the specified TargetRefs (Services), and if a Service matches a BackendTLSPolicy, then
-assume the connection is TLS, and verify that the TargetRef’s certificate can be validated by the client (Gateway) using
-the provided certificates and hostname before the connection is made. On the question of how to signal
-that there was a failure in the certificate validation, this is left up to the implementation to return a response error
-that is appropriate, such as one of the HTTP error codes: 400 (Bad Request), 401 (Unauthorized), 403 (Forbidden), or
+- How to verify the peer certificate, including signing certificate verification and other properties of the certificate (subject alt names).
+  - Support for referencing a CA certificate, as well as a help to use the default trusted system certificates.
+
+Two serve these dual roles, `BackendTLSPolicy` is usable in two contexts:
```

Comment:
Looking from `Ana` perspective, I would like to see a guideline (or strong rules) helping me to choose between setting the policy as a direct Service attachment and HTTP Route override.
For example can Ana define BackendTLSPolicy as Route override for defining Application TLS instead of using direct Service Attachment?  I don't see any constraints in this GEP to stop Ana from doing that and IMO this is sth we need to avoid.